src/validate_pyproject/pyproject_toml.schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
 
-  "$id": "https://packaging.python.org/en/latest/specifications/declaring-build-dependencies/",
+  "$id": "https://packaging.python.org/en/latest/specifications/dependency-specifiers/",
```

Comment:
I see that this new ref fix linkcheck however it is not the best reference.
The new PyPA docs now unify both the `[build-system]` table and the `[project]` table into a single URL: https://packaging.python.org/en/latest/specifications/pyproject-toml/.

We could try to use fragments like in https://packaging.python.org/en/latest/specifications/pyproject-toml/#pyproject-build-system-table, but I have the impression that this will confuse the validation (I might be wrong though).

For the time being we can live as it is, because it is not hurting anyone...
The folks maintaining the docs are currently having a look on what happened to the redirect: https://github.com/pypa/packaging.python.org/issues/1486.

@cclauss is it OK if once they finished debugging the redirects we return the old URL? It is going to be a 30X status, I am not sure if that will fail the linkcheck... Alternatively we can test the fragments and see if they actually break something...

Reply:
Please go ahead and revert the specification changes if you think that is the right thing to do.

The old URLs do not exist (404 error) but those were only warnings, not errors in linkcheck.

The error was that blank line!